### ISSUE TITLE
walkthrough setup.md tf version and geopandas installation error and fix. 

### ISSUE THREAD

**Issue by Zia-** [GIScience/ohsome2label#9]
While trying to setup training env, I came across the following error - 

- python3.6 does not support tf v1.*
- geopandas had some broken file link to gdal (https://stackoverflow.com/questions/54734667/error-installing-geopandas-a-gdal-api-version-must-be-specified-in-anaconda).

The following two steps helped me go through these errors -

- use it to initiate an env with python 2.7 version
`virtualenv -p python2.7 venv`
- install the following two packages while installing _Install System Level Libraries_
`apt-get install libgeos-dev`
`apt-get install libgdal-dev`

Note that for ohsome2label we still need python3.6 version. One can use different virenv for that.

Hope it helps!

**Comment by bobleegogogo** [comment]
@Zia- thanks for reporting this error!

Tips regarding the installation of geopands definitely help. I added them now in 8299921d195c5624ffaff7be16262f0f52228069.

Not sure about "python3.6 does not support tf v1.*", does this mean you cannot "pip install tensorflow==1.14.0" with a python3.6 env? while it did work for me when I wrote the walkthrough. 

However, in general, I need to think about a more up-to-date implementation of TF object detection API.

**Comment by Zia-** [comment]
@bobleegogogo 

> Not sure about "python3.6 does not support tf v1.*", does this mean you cannot "pip install tensorflow==1.14.0" with a python3.6 env? while it did work for me when I wrote the walkthrough.

Yes, from what I understood that there was some python version issue with tf.
For example, tf.session etc is deprecated in version2. So, tfv1.14.0 is needed. However, for me python3.6 couldn't install it. May be it was pip3 issue. Can't say!

For now, my working solution is using two virenv. One with python2.6 dealing with TF_objectdetection and another one with python3.6 dealing with ohsome2label.
